### DISCUSSION TITLE
How to make a limit on saving data to storage?

### DISCUSSION THREAD

**Discussion by Frubl** [GrapesJS/grapesjs#6311] [Q&A]
I want to set a limit of saving data for example for 1 second. How can i do it?

I have stepsBeforeSave: 1, so it's possible to spam with savings and overload the server. I want to prevent it. Is there a simple way of how to do it?

**Comment by danstarns** [comment]
How are you saving the data? You could hook into the events of saving -  See https://grapesjs.com/docs/api/commands.html#available-events and please share something more reproducible 🙏
